client/gtk2/ibusimcontext.c

```diff
@@ -1179,7 +1182,7 @@ ibus_im_context_filter_keypress (GtkIMContext *context,
         GdkModifierType state = gdk_event_get_modifier_state (event);
         if (state & IBUS_HANDLED_MASK)
             return TRUE;
-        if (state & IBUS_IGNORED_MASK)
+        if (state & IBUS_IGNORED_MASK && !(state & IBUS_PREFILTER_MASK))
```

Comment:
I disagree to handle the return value with the configurable flag.

Reply:
Which implementation exactly?